core/util/src/main/java/org/phoebus/util/time/TimeParser.java

```diff
@@ -27,6 +18,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static java.time.temporal.ChronoUnit.*;
```

Comment:
Yes!
IDEs tend to help with imports, so there's usually no extra effort to have explicit imports, and having the explicit import removes any chances of wondering which one it is in case of name clashes.
Are there IDEs which replace explicit imports with wildcard imports when a certain number of imports use the same package? Then I suggest to turn that off and alway use explicit imports.